core/bind/core_bind.cpp

```diff
@@ -559,7 +559,7 @@ Vector<String> _OS::get_cmdline_args() {
 
 String _OS::get_locale() const {
 
-	return OS::get_singleton()->get_locale();
+	return OS::get_singleton()->get_locale().replace("_", "-");
```

Comment:
I thought we were going to do the opposite :slightly_smiling_face: 

Underscore separators seem to be a bit more common in software localization, even if they go against some RFCs I've seen.

Reply:
We standardized on the glibc identifiers with an underscore, yes.
But I'm not sure `get_locale()` should be changed, I think it's good that it gives the system locale. Instead, we should likely add a `OS.get_standardized_locale()` that returns the locale standardized with `_` (and if need be converting things like `zh-Hans` to `zh_CN`, but I wouldn't do it in a first step).